### ISSUE TITLE
Multiple tabs causes cumulative opaqueness 

### ISSUE THREAD

**Issue by VA1DER** [mintty/mintty#1152]
Creating multiple transparent tabs in mintty is causing the opaqueness of them all to accumulate in the window.  The man page states that since version 3.5.1 that this should not be the case.

Version: mintty 3.6.0 (x86_64-pc-cygwin)
OS: Windows 10 64 bit
Command-line: `C:\apps\cygwin64\bin\mintty.exe --tabbar=4 -i /Cygwin-Terminal.ico -`
Config:
```
FontHeight=10
Transparency=medium
Columns=142
Rows=52
BackgroundColour=3,18,29
```

Steps to duplicate:
1. Invoke mintty with above command line and configuration
2. Create a new tab, either by clicking on the program's title-bar icon and selecting "new tab", or invoking a new session
3. Observe that when the new tab is created that the transparency of the underlying Windows desktop is reduced

Expected Behaviour:
> Since 3.5.1, mintty manages tabs in order to support transparency (**_avoiding cumulation of opaqueness_**), hide background tabs from appearing while moving the window, and keep the tabbar consistent in case a tab gets terminated irregularly. [emphasis added]

**Comment by mintty** [comment]
Sorry, that section of the manual is an artefact of a failed attempt to implement that support which was withdrawn for the release :(
Transparency with Virtual Tabs is in fact _not_ supported.

**Comment by VA1DER** [comment]
It's unfortunate that this support is withdrawn.  The transparency itself works fine, just the fact that multiple tabs (windows) are drawn overlapping is what is causing the opacity increase.  Would it not be possible to draw only the window for the active tab and have all others left undrawn?

As a workaround I've just halved my opacity since I almost always have exactly two tabs open.

Thanks for the update, and thanks for the tab support.  It's very useful.

**Comment by mintty** [comment]
>  Would it not be possible to draw only the window for the active tab and have all others left undrawn?

Sure, that's exactly the approach I tried to implement, but it failed so far as it also means steady fighting against Windows APIs. Hiding background windows sounds like an elegant solution, but what if the active window is getting closed? Who is then coming to the rescue to reactivate one of the hidden ones? One major and vastly intractable problem was that hidden windows do not receive any Windows API messages anymore, at which point I suspended the feature.
